src/NuGet.Clients/VisualStudio/NuGet.VisualStudio.csproj

```diff
@@ -75,6 +75,8 @@
     <Compile Include="Extensibility\IVsFrameworkCompatibility.cs" />
```

Comment:
unrelated, but please rename the project to match the generated dll name like any other project